$OpenBSD: patch-Build_PL,v 1.2 2021/01/16 12:15:47 fcambus Exp $

Do not pass the -fms-extensions flag, it is unsupported on OpenBSD and
breaks the build.

Index: Build.PL
--- Build.PL.orig
+++ Build.PL
@@ -14,7 +14,7 @@ my $mb = Module::Build->new(
 
 $mb->extra_compiler_flags(
     @{ $mb->extra_compiler_flags },
-    qw(-std=c99 -fms-extensions -Wall -g)
+    qw(-std=c99 -Wall -g)
 );
 $mb->extra_linker_flags( @{ $mb->extra_linker_flags }, '-lmaxminddb' );
 
